src/tasks/semantic-release/index.js

```diff
@@ -22,7 +17,7 @@ module.exports = function(config) {
 	// package.json
 	const pkg = packageJson();
 
-	if (!hasSemanticRelease()) {
+	if (!pkg.getScript('semantic-release')) {
```

Comment:
Probably we need to short-circuit it in some way. Right now if you run it twice, it will show this weird error.

Reply:
Yeah, but after you run semantic-release again it will go away again so I think that's okay for the first iteration. This task suppose to run only once and I don't have any idea of how to solve this without introducing some state :-/

Reply:
OK, we’ll think about that later ;-)